flipperdevices/flipperzero-firmware
# Picopass load/info/delete

# What's new

- Add loading, info, delete for picopass (heading towards write support)

I wanted to open this for feedback before I headed off to Defcon.  No rush, I probably won't attend to any feedback until I return Monday.

# Verification 

- Launch Picopass
- Choose "Saved"
- Select file
- Choose "Info"
- See info

# Checklist (For Reviewer)
